docs_user/modules/proc_migrating-the-rgw-backends.adoc

```diff
@@ -114,59 +107,36 @@ rgw.rgw.cephstorage-1.qynkan  cephstorage-1  172.17.3.26:8090  running (16s)
 rgw.rgw.cephstorage-2.krycit  cephstorage-2  172.17.3.81:8090  running (13s)
 ----
 
-. Ensure that the new RGW back ends are reachable on the new ports, so you can enable an IngressDaemon on port 8080 later. Log in to each {CephCluster} node that includes RGW and add the `iptables` rule to allow connections to both 8080 and 8090 ports in the {CephCluster} nodes:
+. Ensure that the new RGW back ends are reachable on the new ports, so you can enable an IngressDaemon on port `8080` later. Log in to each {CephCluster} node that includes RGW and add the `iptables` rule to allow connections to both 8080 and 8090 ports in the {CephCluster} nodes:
 +
 ----
-iptables -I INPUT -p tcp -m tcp --dport 8080 -m conntrack --ctstate NEW -m comment --comment "ceph rgw ingress" -j ACCEPT
-
-iptables -I INPUT -p tcp -m tcp --dport 8090 -m conntrack --ctstate NEW -m comment --comment "ceph rgw backends" -j ACCEPT
-
-for port in 8080 8090; {
-    for i in 25 10 32; {
-       ssh heat-admin@192.168.24.$i sudo iptables -I INPUT \
-       -p tcp -m tcp --dport $port -m conntrack --ctstate NEW \
-       -j ACCEPT;
-   }
-}
+$ iptables -I INPUT -p tcp -m tcp --dport 8080 -m conntrack --ctstate NEW -m comment --comment "ceph rgw ingress" -j ACCEPT
+$ iptables -I INPUT -p tcp -m tcp --dport 8090 -m conntrack --ctstate NEW -m comment --comment "ceph rgw backends" -j ACCEPT
 ----
 
 . From a Controller node, such as `controller-0`, try to reach the RGW back ends:
 +
 ----
-for i in 26 23 81; do {
-    echo "---"
-    curl 172.17.3.$i:8090;
-    echo "---"
-    echo
-done
+$ curl http://cephstorage-0.storage:8090;
 ----
 +
 You should observe the following output:
 +
 ----
----
-Query 172.17.3.23
-<?xml version="1.0" encoding="UTF-8"?><ListAllMyBucketsResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><Owner><ID>anonymous</ID><DisplayName></DisplayName></Owner><Buckets></Buckets></ListAllMyBucketsResult>
----
-
----
-Query 172.17.3.26
-<?xml version="1.0" encoding="UTF-8"?><ListAllMyBucketsResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><Owner><ID>anonymous</ID><DisplayName></DisplayName></Owner><Buckets></Buckets></ListAllMyBucketsResult>
----
-
----
-Query 172.17.3.81
 <?xml version="1.0" encoding="UTF-8"?><ListAllMyBucketsResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><Owner><ID>anonymous</ID><DisplayName></DisplayName></Owner><Buckets></Buckets></ListAllMyBucketsResult>
----
 ----
 
+Repeat the verification for each node where a RGW daemon is deployed.
```

Comment:
```suggestion
----
+
Repeat the verification for each node where a RGW daemon is deployed.
```

You need the plus sign to align the text under the code block.

Reply:
Done (thank you)